macos/FluentUISwiftUITestApp/TestViewControllerWrappingView.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import AppKit
+import FluentUITestViewControllers
+import SwiftUI
+
+struct TestViewControllerWrappingView: NSViewControllerRepresentable {
+	
+	var type: NSViewController.Type
+	
+	func makeNSViewController(context: Context) -> some NSViewController {
+		return type.init(nibName: nil, bundle: nil)
+	}
+	
+	func updateNSViewController(_ nsViewController: NSViewControllerType, context: Context) {
+	}
```

Comment:
I'm guessing this is an override and that's why it has to exist? If so, why isn't there the override keyword?